tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
+System.Runtime.WindowsRuntime.UI.Xaml.dll
```

Comment:
@parjong  I was think you might be able to identify someone to populate the arm/x86 lists if needed.